src/PerformanceTests/Persistence.V6.Sql.V2/SqlProfile.cs

```diff
@@ -16,4 +21,14 @@ public void Configure(EndpointConfiguration configuration)
         var subscriptions = persistence.SubscriptionSettings();
         subscriptions.CacheFor(TimeSpan.FromSeconds(1));
     }
+
+    void ISetup.Setup()
+    {
+        var cs = ConfigurationHelper.GetConnectionString(Permutation.Persister.ToString());
+        var assembly = Assembly.GetExecutingAssembly();
```

Comment:
Should this three lines by some kind of a helper?

Reply:
Jups, now used multiple times. I'll refactor that.